hubblestack/extmods/modules/pulsar.py

```diff
@@ -958,20 +977,30 @@ def top(topfile='salt://hubblestack_pulsar/top.pulsar',
 
 
 def get_top_data(topfile):
+    '''
+    Cache the topfile and process the list of configs this host should use.
+    '''
+    # Get topdata from filesystem if we don't have them already
+    global TOP
+    global TOP_STALENESS
+    if TOP and TOP_STALENESS < 60:
```

Comment:
You decide - if things break, go even less than once a minute? (like every 5 minutes) Not a huge deal. Just making sure it won't fill the logs too much.

Reply:
As we discussed in slack, this is a very lightweight operation, and the only reason I even track staleness is to keep it from logging too often.